packages/winpanda/extra/src/winpanda/docs/README.md

```diff
@@ -1,4 +1,139 @@
-# Winpanda Design Documents
+# Winpanda The DC/OS Windows Package Manager
+
+# Major features
+
+# Documentation
+
+## Command line parametres
+
+
+ ### Usage:
+ ```
+  winpanda {setup} [options]
+  winpanda {teardown} [options]
+  winpanda {start} [options]
+  winpanda {stop} [options]
+```
+
+### Commands:
+
+* Setup - This command  doing following in sequence.
+    1. Reads cluster.conf from etc folder of inst_root looks for bootstrap node configured there.
```

Comment:
Use single back-quote around identifier names like `cluster.conf`, `etc`, `inst_root`. This applies throughout.